yarn.lock

```diff
@@ -942,9 +942,9 @@ debug@~4.3.1, debug@~4.3.2:
     ms "2.1.2"
 
 decode-uri-component@^0.2.0:
```

Comment:
```suggestion
decode-uri-component@^0.2.2:
```

Reply:
That's not really required. Anything `0.2.x` is fine.